standard/interfaces.md

```diff
@@ -464,7 +463,8 @@ This clause augments the description of indexers in classes [§15.9](classes.md#
 Interface indexers are declared using *indexer_declaration*s ([§15.9](classes.md#159-indexers)), with the following additional rules:
 
 - *indexer_modifier* shall not include `override`.
-- An *indexer_declaration* that has an *expression body* or contains an accessor with a body is an implementation ([§19.1](interfaces.md#191-general)), so it is *not* abstract.
+- An *indexer_declaration* that has an *expression body* or contains an accessor with a body is `virtual`; the `virtual` modifier is not required, but is allowed.
+- An *indexer_declaration* whose accessors are semi-colons (`;`) is `abstract`; the `abstract` modifier is not required, but is allowed.
```

Comment:
I would say that the accessor is `get;`, not that the accessor is `;`. The accessor has a semi-colon. Per the previous comment, could we adopt the lingo here of "whose accessors have no bodies"?

Reply:
@jnm2 – I understand where you’re coming from here, and most people probably think of a semicolon as “no body”. It would be great if indeed there was no body, but for syntax reasons there is – a semicolon.

To change I think wherever there is a semicolon body in the grammar it would need to change to something stating that this is referred to as having no body, and in other places state that having a block or expression is referred to as having a body, etc. 

it is a bit ugly as is, but it is precise, is it worth changing?

If you do decide to do a PR (post 8/pre 9?) I’ll agree to review, and I'll reserve my answer to whether it is worth it till then ;-)

Reply:
Opened https://github.com/dotnet/csharpstandard/issues/1430 to track this suggestion further.